### ISSUE TITLE
Feature Request: Recon after superman installation and updates

### ISSUE THREAD

**Issue by Guisch** [Macjutsu/super#176]
Hey, thanks for your work on superman.

I'm deploying through Jamf. Documentation says it's not recommended to run a recon (jamf inventory) in the same policy as superman.
I know superman already run recon after a macOS update/upgrade. However, I would like to have superman running a recon after superman installation (first superman install or superman update).

Thanks !

**Comment by Macjutsu** [comment]
I will not be adding this feature to `super` because the choice to run a Recon at the end of a Jamf Pro Policy is best handled via Jamf Pro directly.

However, I have updated the Wiki to clarify the issue. The new text reads...

"Avoid adding an inventory collection to your `super` Policy if you have also enabled the "Collect available software updates" option in the Jamf Pro inventory collection settings. This is to prevent an issue where the softwareupdate process hangs when multiple processes (the jamf binary and super) are simultaneously asking for a list of updates."

In other words, it's fine to include a Recon in your `super` deployment Policy as long as you DON'T also have the "Collect available software updates" option enabled. Given that the Jamf Pro Patch Management titles are a significantly better method to track available updates, you shouldn't have this inventory collection option enabled anyhow. Then, you can run as many Recons as you want without interfering with `softwareupdate` or `super`.

**Comment by Guisch** [comment]
Yep that's what we did in the end. Thanks !
